l2tdevtools/build_helpers/dpkg.py

```diff
@@ -274,11 +286,17 @@ def _RemoveOlderOriginalSourcePackage(self, project_name, project_version):
     """
     filenames_to_ignore = '^{0:s}_{1!s}.orig.tar.gz'.format(
         project_name, project_version)
+    if version_suffix and distribution:
+      filenames_to_ignore = '{0:s}_{1:s}{2:s}~{3:s}.orig.tar.gz'.format(
+          project_name, project_version, version_suffix, distribution)
     filenames_to_ignore = re.compile(filenames_to_ignore)
 
     # Remove files of previous versions in the format:
     # project_version.orig.tar.gz
     filenames_glob = '{0:s}_*.orig.tar.gz'.format(project_name)
```

Comment:
I opt to make this an if-else instead of an set-if-override

Reply:
Done